src/Nox.Ui.Blazor.Lib/Components/NoxTypes/EditMonth.razor.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Microsoft.AspNetCore.Components;
+using Nox.Types;
+using Nox.Ui.Blazor.Lib.Models;
+using System.Globalization;
+
+namespace Nox.Ui.Blazor.Lib.Components.NoxTypes;
+
+public partial class EditMonth : ComponentBase
+{
+
+    #region Declarations
+
+    [Parameter]
+    public byte? Month { get; set; }
+
+    [Parameter]
+    public CultureInfo CultureInfo { get; set; } = CultureInfo.CurrentCulture;
+
+    [Parameter]
+    public string? Title { get; set; }
+
+    [Parameter]
+    public EventCallback<byte?> MonthChanged { get; set; }
+
+    public static Dictionary<byte, string?> MonthSelectionList { get; set; } = new Dictionary<byte, string?>();
```

Comment:
readonly and maybe not nullable string

Reply:
sorry cant get the readonly to work it says not valid - same in Edittime as well